youcan-shop/cod-theme-3
# YSHOP2-511: Logo should default to YouCan logo

## JIRA Ticket

[YSHOP2-511](https://youcanshop.atlassian.net/browse/YSHOP2-511)

## QA Steps

-  [ ] Remove logos from the footer and navbar, and you should see the YouCan logo

## Note

Leave empty when you have nothing to say


[YSHOP2-511]: https://youcanshop.atlassian.net/browse/YSHOP2-511?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
